modules/argocd/Makefile

```diff
@@ -1,15 +1,31 @@
-RELEASE_TAG ?= 0.1.0
-.PHONY: updater/test check-prereq-variables
+RELEASE_TAG ?= 1.0.0
+dirs ?= rendered/
+env ?=
+app ?= ""
+
+dev_dir = rendered/environments/aws.dev
+qa_dir = rendered/environments/aws.qa
+logs_dir = rendered/environments/aws.logs
+prod_dir = rendered/environments/aws.prod
+
+.PHONY: updater/test check-prereq-variables get-dirs
 
 ## Test argocd-image-updater annotations
-updater/test: check-prereq-variables
+updater/test: check-prereq-variables get-dirs
```

Comment:
I think this module should sit under `k8s/argocd` which is then more consistent with our other helpers.

I also think `updater` is a bit too generic so would suggest `image-updater`

So you would end up with 

```
make k8s/argocd/image-updater/...
```